plugins/leemons-plugin-leebrary/frontend/src/helpers/uploadFileAsMultipart.js

```diff
@@ -177,9 +177,13 @@ async function getZipFiles(jsfile) {
   return Promise.all(_.map(entries, downloadEntry));
 }
 
-async function uploadFileAsMultipart(jsfile, { onProgress = () => {}, name } = {}) {
+async function uploadFileAsMultipart(
+  jsfile,
+  { onProgress = () => {}, name } = {},
+  isScormFolder = false
+) {
   if (jsfile instanceof File || jsfile instanceof Blob) {
-    const isFolder = jsfile.name?.endsWith('.zip');
+    const isFolder = isScormFolder && jsfile.name?.endsWith('.zip');
```

Comment:
I would rather use isFolder as param to allow new plugins to save zips as folders.

Although I receive this isFolder I would keep checking if it's a zip to avoid errors inside of the funcion

Reply:
And I would keep the property in the params object for more verbosity in the call